pkgs/os-specific/linux/systemd/default.nix

```
@@ -231,17 +231,11 @@ stdenv.mkDerivation (finalAttrs: {
     ./0016-systemctl-edit-suggest-systemdctl-edit-runtime-on-sy.patch
     ./0017-meson.build-do-not-create-systemdstatedir.patch
 
-    # https://github.com/systemd/systemd/pull/33258
-    # Remove after 256.3
-    (fetchpatch {
-      url = "https://github.com/systemd/systemd/compare/b268a71069786a45460807967e669d505ba3c5a2..f26b2ec46118a4493608618da2253bb9dfc6b517.patch";
-      hash = "sha256-OmuPDm3NykrDeNTA3NcYt9iTXEUFwKJ5apPP4KqtABg=";
-    })
-
-    # https://github.com/systemd/systemd/pull/33400
-    (fetchpatch {
-      url = "https://github.com/systemd/systemd/compare/051d462b42fe6c27824046c15cd3c84fa5afe05b..5e2d802c018f0b6d5dd58745f64d6958fa261096.patch";
-      hash = "sha256-drGAnx+ECixOjIP0DUSbCG/emUgoVips9WQL5ny3NKQ=";
+    # https://github.com/systemd/systemd/issues/33392
+    (fetchpatch2 {
+      url = "https://github.com/systemd/systemd/commit/f8b02a56febf14adf2474875a1b6625f1f346a6f.patch";
```

```suggestion
      url = "https://github.com/systemd/systemd/commit/f8b02a56febf14adf2474875a1b6625f1f346a6f.patch?full_index=1";
```

(and hash ofc would change)

ref https://github.com/NixOS/nixpkgs/issues/257446#issuecomment-1736563091

---

ah, thanks

---

@eclairevoyant Oddly enough, it looks like the hash is unchanged, as the revisions are stripped out completely. I believe that's because of `revert = true;`. Oh well, added it anyway as a good practice.